index.html

```diff
@@ -1,19 +1,74 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
-
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
-
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <title>Karma</title>
+        <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/style.css">
+        <link rel="stylesheet" href="store.html">
+        <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    </head>
+    <body>
+        <header>
+            <nav>
+                <img class="logo" src="img/karma-logo.svg" alt="logo">
+                <a href="" class="a-options">Login</a>
+                <a href="" class="a-options">Help</a>
+                <a href="" class="a-options">Blog</a>
+                <a href="store.html" class="a-options">Store</a>
+                <a href="" class="a-options">How it Works</a>
+                <a href="index.html" class="a-options"><strong>Meet Karma</strong></a>
+            </nav>
+        </header>
+        <main>
+            <div class="container">
+                <img class="bg-image" src="img/first-background.jpg" alt="bg-img">
+                <div class="bg-text">
+                    <p class="p-1">Introducing Karma</p>
```

Comment:
This is a title and needs to be different from "Bring wifi ... " text. I highly suggest to use h1 or h2 , so users can have a better experience.